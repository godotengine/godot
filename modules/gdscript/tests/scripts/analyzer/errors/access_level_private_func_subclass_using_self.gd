@private

class Foo:
    func foo():
        pass

class Bar extends Foo:
    func bar():
        self.foo()

func test():
    pass
