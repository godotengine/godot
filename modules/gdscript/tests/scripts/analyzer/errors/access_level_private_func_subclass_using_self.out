GDTEST_ANALYZER_ERROR
Cannot call private function "foo()" on the class Bar.
